src/chaosinventory/settings.py

```diff
@@ -1,8 +1,11 @@
 import configparser
 import os
+import logging
 from pathlib import Path
 
 config = configparser.RawConfigParser()
+# Read fallback config, initalizing all values needed for basic execution
+config.read_file(open(Path(__file__).parent / "fallback.cfg"))
```

Comment:
With the changes in https://github.com/chaosinventory/chaosinventory/pull/41/commits/98eea83c2edfbec43e75091c5eb38e53a45e6c2b, the fallback configuration file is out of scope of this pull request.